src/DynamoCore/Graph/Workspaces/SerializationConverters.cs

```diff
@@ -107,11 +108,16 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
                type = Type.GetType(obj["$type"].Value<string>());
                typeName = obj["$type"].Value<string>().Split(',').FirstOrDefault();
 
-               if (typeName.Contains("ZeroTouch"))
-               {
-                    // This assemblyName does not usually contain version information...
+                if (typeName.Contains("ZeroTouch"))
+                {
+                    // If it is a zero touch node, then get the whole function name including the namespace.
                     functionName = obj["FunctionSignature"].Value<string>().Split('@').FirstOrDefault().Trim();
```

Comment:
do FunctionSignatures always have the `@` token? What if it's not there, what happens?